scalableminds/webknossos
# Fix guessed dataset boundingbox for non-zero-aligned datasets

### Steps to test:
- have a dataset without a 0,0,0-cube
- remove dataset-properties.json
- click import
- guessed bounding box should not be too large

### Issues:
- fixes #3435 

------
